jormungandr-lib/src/interfaces/reward_parameters.rs

```diff
@@ -98,6 +128,11 @@ mod test {
                     epoch_start: Epoch::arbitrary(g),
                     epoch_rate: NonZeroU32::new(Arbitrary::arbitrary(g))
                         .unwrap_or(NonZeroU32::new(1).unwrap()),
+                    rewards_limit: Ratio::arbitrary(g),
+                    npools: NonZeroU32::new(Arbitrary::arbitrary(g))
+                        .unwrap_or(NonZeroU32::new(1000).unwrap()),
+                    npools_threshold: NonZeroU32::new(Arbitrary::arbitrary(g))
+                        .unwrap_or(NonZeroU32::new(100).unwrap()),
```

Comment:
why `100` ?

Reply:
This will allow 100 pools before any penalty is applied.  This seemed to be the initial target number of pools.  Again this will confirm that the mechanisms are working.

Reply:
this is for the test. `1` is what has been used everywhere else to replace the `0` case.

Reply:
See above.  Needs to be at least 2 and preferably higher to exercise the test.